pca_ica/run_pca.m

```diff
@@ -6,24 +6,72 @@ function run_pca(movie_source, num_PCs)
 %   movie_source: Movie filename
 %   num_PCs: Number of requested PCs
 %
+% Variable input arguments:
+%   'trim': if added as an argument then the script trims pixels in the 
+%       movie that do not cross above the median of the maximum pixel 
+%       values.
+%   'medfilt': Add it as an argument to perform median-filtering on the 
+%       movie on a per-frame basis before PCA. Highly recommended since
+%       especially the miniscope camera might have various artifacts.

```

Comment:
I think you can leave out the "Highly recommended since..." in this comment block.